internal/cli/cli.go

```diff
@@ -60,11 +60,15 @@ type deleteOpts struct {
 	confirm        bool
 	successMessage string
 	failMessage    string
+	authDB         string
```

Comment:
this should not be here, not all deleters should have an `authDB`, this should work the same as a project id that is passed as a parameter of the function